.github/workflows/e2e-detox-pr.yml

```diff
@@ -10,7 +12,7 @@ on:
 jobs:
   update-initial-status:
     runs-on: ubuntu-22.04
-    if: contains(github.event.pull_request.labels.*.name, 'E2E iOS tests for PR')
+    if: contains(join(github.event.pull_request.labels.*.name, ','), 'E2E iOS tests for PR')
```

Comment:
The behaviour in that case depends on the `concurrency` directive in the workflow: in case you set the two labels one after the other, my suggested concurrency would terminate the oldest of the two workflows, and just run the most recent one.

This would indeed leave around the older of the two labels, but without significant downsides.

I have updated [this suggestion](https://github.com/mattermost/mattermost-mobile/pull/8202#discussion_r1777494108) to further optimize this, e.g. to make it possible to run the normal E2E tests simultaneously to the throttled E2E test.

Reply:
I believe we're going to change something either way (in this file or somewhere else) whenever there will be a new LBW configuration. But this also works fine :+1: